Lang/Spanish.pj.Lang

```diff
@@ -419,7 +420,7 @@
 //Edit Cheat
 #1027#  "Editar Truco"
 #1028#  "Actualizar Truco"
-#1029#  "El Truco ha sido cambiado.\n\n¿Deseas actualizar?"
+#1029#  "El Truco ha sido cambiado.\n\n¿Quieres actualizar?"

```

Comment:
"deseas" is more used for a choice related with a pleasure question, for example: a food or sex.

"quieres" is more used for a choice related with a common question, for example: an action, a step.